packages/circle-demo-webapp/tailwind.config.ts

```diff
@@ -15,7 +15,6 @@ export default {
         sm: 'calc(var(--radius) - 4px)',
       },
       colors: {
-        error: '#D66262',
```

Comment:
I updated color of destructive to be red-500 and use that class everywhere with errors, so `border-destructive` or `text-destructive`